app/grandchallenge/github/models.py

```diff
@@ -109,15 +110,33 @@ def tag(self):
     def tag_url(self):
         return f"https://github.com/{self.payload['repository']['full_name']}/releases/tag/{self.payload['ref']}"
 
+    @property
+    def user_error(self):
+        if "This repository is over its data quota" in self.stdout:
+            return (
+                f"Repository {self.repo_name} has used all of its LFS "
+                f"bandwidth. Please purchase more data packs on GitHub for "
+                f"this repo so that we can clone it."
+            )
+        else:
+            return ""
+
     def save(self, *args, **kwargs):
-        adding = self._state.adding
+        if self._state.adding:
+            if self.payload.get("ref_type") == "tag":
+                on_commit(
+                    lambda: get_zipfile.apply_async(kwargs={"pk": self.pk})
+                )
+            elif self.payload.get("action") == "deleted":
+                on_commit(
+                    lambda: unlink_algorithm.apply_async(
+                        kwargs={"pk": self.pk}
+                    )
+                )
```

Comment:
I checked the database and there is no message with the combination of these options in the payload, so it is safe to make this if statement an elif IMO.